website/docs/docs/deploy/slim-ci-jobs.md

```diff
@@ -0,0 +1,140 @@
+         <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Enabling-CI/repo-config.png" alt="Configure repo"/>
+         <br></br>
+         Select the <b>GitHub</b>, <b>GitLab</b>, or <b>AzureDevOps</b> tab and reselect your repository. That should complete the setup of the project and enable you to set up a dbt Cloud CI job.</div>
+   </div>
+</details>
+<details>
+   <summary>Error messages that refer to schemas from previous PRs</summary>
+   <div>
+      <div>If you receive a schema-related error message referencing a <i>previous</i> PR, this is usually an indicator that you are not using a production job for your deferral and are instead using <i>self</i>.  If the prior PR has already been merged, the prior PR's schema may have been dropped by the time the Slim CI job for the current PR is kicked off.<br></br> <br></br>
+      
+      To fix this issue, select a production job run to defer to instead of self.
+      </div>
+   </div>
+</details>
+<details>
+   <summary>Production job runs failing at the <b>Clone Git Repository</b> step</summary>
+   <div>
+      <div>dbt Cloud can only checkout commits that belong to the original repository. dbt Cloud _cannot_ checkout commits that belong to a fork of that repository.<br></br><br></br>
+      
+      If you receive the following error message at the <b>Clone Git Repository</b> step of your job run:<br></br>
+         <code>
+         Error message:<br></br>
+         Cloning into '/tmp/jobs/123456/target'...<br></br>
+         Successfully cloned repository.<br></br>
+         Checking out to e845be54e6dc72342d5a8f814c8b3316ee220312...<br></br>
+         Failed to checkout to specified revision.<br></br>
+         git checkout e845be54e6dc72342d5a8f814c8b3316ee220312<br></br>
+         fatal: reference is not a tree: e845be54e6dc72342d5a8f814c8b3316ee220312<br></br>
+         </code><br></br><br></br>
+         
+         Double-check that your PR isn't trying to merge using a commit that belongs to a fork of the repository attached to your dbt project.</div>
+   </div>
+</details>
+<details>
+   <summary>CI job not triggering for Virtual Private dbt users</summary>
+   <div>
+      <div>To trigger jobs on dbt Cloud using the <a href="https://docs.getdbt.com/docs/dbt-cloud-apis/admin-cloud-api">API</a>, your Git provider needs to connect to your dbt Cloud account.<br></br><br></br>
+      
+      If you're on a Virtual Private dbt Enterprise plan using security features like ingress PrivateLink or IP Allowlisting, registering CI hooks may not be available and can cause the job to fail silently.</div>
+   </div>
+</details>
+
+### Temp PR schema limitations
+
+If your temporary pull request schemas aren't dropping after a merge or close of the PR, it's likely due to the below scenarios. Open and review the toggles below for recommendations on how to resolve this:
```

Comment:
How will they know their temporary PR schemas aren't dropping? Would they see them linger in the Cloud UI? If so where? AN example might be helpful.